src/components/AlbumActions.vue

```diff
@@ -100,6 +100,14 @@ export default {
         newAlbum: { review_comment: reviewComment },
       });
     },
+    filterTracks: function () {
```

Comment:
Write this as a filter + map, that's much clearer. I also don't think this needs to be a separate function, the filter plus map operation is easy enough. (Or at least the function needs to be named better.)